src/server/api/routers/github-router.ts

```diff
@@ -0,0 +1,82 @@
+import { z } from "zod";
+import { App } from "octokit";
+import { readFileSync } from "fs";
+
+import {
+    createTRPCRouter,
+    protectedProcedure,
+    publicProcedure,
+} from "~/server/api/trpc";
+
+// GitHub router
+export const gitHubRouter = createTRPCRouter({
+    hello: publicProcedure
+        .input(z.object({ text: z.string() }))
+        .query(({ input }) => {
+            return {
+                greeting: `Hello ${input.text}`,
+            };
+        }),
+
+    // Webhook endpoint
+    makeWebhook: protectedProcedure
```

Comment:
Might be worth pulling out the contents of router into a separate function for easier testing/isolation.